frontend/packages/noobaa-storage-plugin/src/components/data-consumption-card/data-consumption-card-utils.ts

```diff
@@ -0,0 +1,94 @@
+import {
+  humanizeDecimalBytesWithoutB,
+  humanizeBinaryBytesWithoutB,
+} from '@console/internal/components/utils';
+
+/* Chart Data Handlers */
+
+export const iopsChartData = (data, key) => {
```

Comment:
all these data parsing functions looks very similar to `getRangeVectorStats` and `getInstantVectorStats` from `graphs/utils.ts`. You should reuse them

Reply:
thanx for telling this, but still for IOPS I need a modified function.

Reply:
actually it is not valid for my case, the data coming from prom for each dropdown selection is different. I tried to incorporate these but it did not work out.
@rawagner